README.md

```diff
@@ -119,7 +119,7 @@ Optional parameters are:
 - "env": An array of environment variables for the build (e.g. `env : [ [ name : 'name1', value : 'value1' ], [ name : 'name2', value : 'value2' ] ]`.
 - "showBuildLogs":  Pipe the build logs from OpenShift to the Jenkins console.
 - "checkForTriggeredDeployments":  Verify whether any deployments triggered by this build's output fired.
-- "waitTime":  Time in milliseconds to wait for build completion.  Default is 5 minutes.
+- "waitTime":  Time in milliseconds to wait for build completion.  Default is 15 minutes.
```

Comment:
@oatmealraisin 's pull will be changing the unit to seconds.  FYI another potential rebase it for one of y'all.